cmd/tools/cli/main.go

```diff
@@ -27,12 +27,21 @@ package main
 import (
 	"os"
 
+	"github.com/temporalio/tctl/pkg/config"
 	"go.temporal.io/server/tools/cli"
+	"go.temporal.io/server/tools/cli_next"
 )
 
 // Start using this CLI tool with command
 // See temporal/tools/cli/README.md for usage
 func main() {
-	app := cli.NewCliApp()
-	_ = app.Run(os.Args)
+	version, _ := config.Get("version")
+
+	if version == "next" {
```

Comment:
`v2` ? `next` seems a little bit weird

Reply:
so basically we don't want to introduce another "version" term because this one is not going to stay for long. As we enable "next" by default and later even remove the current tctl completely, it's going to be just "tctl" with a version the same as server 

we've spoke about this with Shawn and Ryland and that's how came up to "next" instead of version numbers